Documentation/ceph-cluster-crd.md

```diff
@@ -284,6 +284,59 @@ For more information on resource requests/limits see the official Kubernetes doc
   - `cpu`: Limit for CPU (example: one CPU core `1`, 50% of one CPU core `500m`).
   - `memory`: Limit for Memory (example: one gigabyte of memory `1Gi`, half a gigabyte of memory `512Mi`).
 
+### Ceph config overrides
+Users are advised to use Ceph's CLI or dashboard to configure Ceph; however, some users may want to
+specify some settings they want set in the `CephCluster` manifest. This is possible using the
+`configOverrides` section. Config overrides follow the same syntax as [Ceph's documentation for
+setting config values](https://docs.ceph.com/docs/master/rados/configuration/ceph-conf/#commands),
+and configuration is stored in Ceph's centralized configuration database.
+
+Each item in `configOverrides` has `who`, `option`, and `value` properties. `who` can be `global` to
+affect all Ceph daemons, a daemon type, an individual Ceph daemon, or a glob matching multiple
+daemons. It may also use a
+[mask](https://docs.ceph.com/docs/master/rados/configuration/ceph-conf/#sections-and-masks).
+`option` is the configuration option to be overridden, and `value` is the value to set on the
+configuration option. All properties are strings.
+
+Rook will only set configuration overrides once when the operator starts (or restarts).This means
+that users can change Ceph's configuration from the CLI or dashboard as desired without Rook
+constantly fighting to control the setting. This is particularly valuable in situations where values
+need to be changed to debug errors or test changes to tuning parameters. Remember that Rook will set
```

Comment:
This `Remember that Rook will set
these overrides any time the operator restarts, so user changes via Ceph's CLI or dashboard to
values set here will not persist forever.` makes sense but I wonder if it shouldn't have a "Warning" flag in the doc for better visibility.